pkg/addon/agent/agent.go

```diff
@@ -84,9 +84,20 @@ func (p *proxyAgent) Manifests(managedCluster *clusterv1.ManagedCluster, addon *
 		proxyServerLoadBalancer = entrySvc
 	}
 
+	// get proxy client secret
+	proxyClientSecret, err := p.nativeClient.CoreV1().
```

Comment:
can you elaborate the case why we need to persist the proxy-client secret to the managed clusters? is that for communication between the managed clusters? anyway, i think we should separate this change into another pull for further expansion, thx

Reply:
i will split the submission of secret in a moment. this secret is for communication between the `proxy-agent` and the `proxy-server`.